src/Microsoft.FeatureManagement/FeatureFilters/Crontab/CrontabField.cs

```diff
@@ -0,0 +1,242 @@
+            // Step values can be used in conjunction with ranges after a slash.
+            string[] segments = content.Split(',');
+            foreach (string segment in segments)
+            {
+                if (segment == null)
+                {
+                    message = InvalidSyntaxErrorMessage(content);
+                    return false;
+                }
+
+                int value = TryGetNumber(segment);
+                if (IsValueValid(value))
+                {
+                    _bits[ValueToIndex(value)] = true;
+                    continue;
+                }
+
+                if (segment.Contains('-') || segment.Contains('*')) // The segment might be a range.
+                {
+                    string[] parts = segment.Split('/');
+                    if (parts.Length > 2) // multiple slashs
+                    {
+                        message = InvalidSyntaxErrorMessage(content);
+                        return false;
+                    }
+
+                    int step = parts.Length == 2 ? TryGetNumber(parts[1]) : 1;
+                    if (step <= 0) // invalid step value 
+                    {
+                        message = InvalidValueErrorMessage(content, parts[1]);
+                        return false;
+                    }
+
+                    string range = parts[0];
+                    int first, last;
+                    if (string.Equals(range, "*")) // asterisk represents unrestricted range
+                    {
+                        (first, last) = (_minValue, _maxValue);
+
+                        if (_kind == CrontabFieldKind.DayOfWeek) // Corner case for Sunday: both 0 and 7 can be interpreted to Sunday
+                        {
+                            last = _maxValue - 1;
+                        }
+                    }
+                    else // range should be defined by two numbers separated with a hyphen
+                    {
+                        string[] numbers = range.Split('-');
+                        if (numbers.Length != 2)
+                        {
+                            message = InvalidSyntaxErrorMessage(content);
+                            return false;
+                        }
+
+                        (first, last) = (TryGetNumber(numbers[0]), TryGetNumber(numbers[1]));
+
+                        if (!IsValueValid(first) || !IsValueValid(last))
+                        {
+                            message = InvalidValueErrorMessage(content, range);
+                            return false;
+                        }
+
+                        if (_kind == CrontabFieldKind.DayOfWeek && last == 0 && last != first) // Corner case for Sunday: both 0 and 7 can be interpreted to Sunday
+                        {
+                            last = 7; // Mon-Sun should be intepreted to 1-7 instead of 1-0
+                        }
+                    }
+
+                    for (int num = first; num <= last; num += step)
```

Comment:
Related to other comment on unit test, but are we intentionally ignoring `31-1/100` with this logic?

Reply:
Yes. This is the behavior of NCronTab when it is parsing "* * 31-1/100 * *".